### DISCUSSION TITLE
ccf-validation-tools/Mammary_Gland/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#290] [General]
# ccf-validation-tools/Mammary_Gland/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Mammary_Gland/comments/

**Comment by aleixpuigb** [comment]
**New terms**
Breast-Mammary-gland : luminal adaptive secretory precursor cell of mammary gland' (CL:4033057)
Breast-Mammary-gland : luminal hormone-sensing cell of mammary gland' (CL:4033058)
Breast-Mammary-gland : lactocyte type 1' (CL:4033059)
Breast-Mammary-gland : lactocyte type 2' (CL:4033060)
Breast-Mammary-gland : lactocyte (CL:0002325)
